content/en/bits_ai/bits_ai_dev_agent/_index.md

```diff
@@ -0,0 +1,133 @@
+
+## Key capabilities 
+
+The following sections detail how Bits AI Dev Agent integrates with Datadog products to generate contextual code fixes.
+
+### Error tracking
+
+In [Error Tracking][1], Bits AI Dev Agent diagnoses and remediates code issues with context and unit-tested fixes:
+- Determines whether an error can be fixed through code and generates a fix with unit tests.
+- Provides links within the chat to relevant files and methods for streamlined navigation.
+- Analyzes errors asynchronously as they arrive.
+- Marks errors with a **Fix available** status and enables filtering to surface those issues.
+
+[Auto-push](#auto-push) is available for this feature. 
+
+### Flaky test management 
+
+Bits AI Dev Agent fixes flaky tests that are detected through Flaky Test Management in [Test Optimization][4] and attempts to verify that tests remain stable.
+
+[Auto-push](#auto-push) is available for this feature. 
+
+### Trace investigation
+
+Bits AI Dev Agent debugs errors and latency directly from [traces][11] using natural language queries:
+- Analyzes and summarizes large traces.
+- Determines likely root causes for errors and latency.
+- Generates code fixes when prompted.
+
+### Product recommendations
+
+Bits AI Dev Agent applies automated code changes based on Datadog insights such as CCM Recommendations, APM Recommendations, and Profiling Insights.
+
+### Code security
+
+Bits AI Dev Agent remediates vulnerabilities at scale, from single issues to large backlogs. You can:
+- Create PR batches to fix multiple vulnerabilities at once.
+- Use the Campaign tool to push PRs incrementally and manage review workload across teams.
+
+### Pull request assistance
+
+Bits AI Dev Agent integrates with GitHub to create pull requests, respond to comments, update commits, and fix CI failures. 
+
+- Generates PR titles and descriptions based on your PR template.
+- Opens PRs as drafts, iterates using CI logs, and marks them ready for review when checks pass.
+- Continues iterating in response to chat messages and review feedback.
+  **Note**: Comment `@Datadog` to prompt Bits for updates to the PR. Bits Dev never auto-merges PRs.
+
+Go to **Bits AI** > **Dev Agent** > **[Code sessions][7]** to see all Dev Agent code sessions and generated PRs. You can search sessions and filter by service, product source, and status.
+
+Learn more about [PR automation and CI iteration][9].
+
+### Auto-push
+
+Auto-push allows the Agent to create branches, push code, and open PRs when it detects something it can help you with. For example, the Agent can:
+- Auto-create PRs for high impact errors (such as 500s or crashes)
+- Update PRs in response to your comments in GitHub
+- Update PRs to address CI failures
+
+When auto-push is disabled, you must review code in Datadog before it gets pushed.
+
+Auto-push is available for Error Tracking and Test Optimization. 
```

Comment:
Might be worth putting the Auto-push section above these two sections? Right now, when they say auto-push is available, it doesn't mean a lot, but it could be an exciting thing if users understood what auto-push is before scrolling down to this section.